draft-ietf-taps-impl.md

```diff
@@ -122,7 +124,23 @@ informative:
         ins: S. Gjessing
         name: Stein Gjessing
     date: 2018-04-15
-
+    NEATPy:
+      title: "Transport Services: A Modern API for an Adaptive Internet Transport Layer"
+      seriesinfo: IEEE Communications Magazine, April 2021
+      author:
+      -
+        ins: M. Welzl
+        name: Michael Welzl
+      -
+        ins: S. Islam
+        name: Safiqul Islam
+      -
+        ins: M. Gundersen
+        name: Michael Gundersen
+      -
+        ins: A. Fischer
+        name: Andreas Fischer
+      date: 2021
```

Comment:
I think this reference is no longer used, and should be removed.

Reply:
ACK, will do this in the new & clean PR that replaces this one.